### DISCUSSION TITLE
Delayed docker images for release

### DISCUSSION THREAD

**Discussion by derdeagle** [kimai/kimai#2877] [Questions & Answers] [answered]
Hi,

I wanted to ask whether it is intentional that (as of now) for some releases there is no docker image available. The latest image version on Docker hub currently available is 1.15.2 while 1.15.6 is the latest release.
Are there currently any blocker to release new images or is it a matter of time where assistance is needed?
I'll open this discussion because I wasn't able to find something related via the search (sorry if I missed it).

Thank you!

**Comment by kevinpapst** [comment, marked as answer]
Hop over to https://github.com/tobybatch/kimai2 and post the same question. 

Would be great if we would get an automated build system at some point... but until then, we can try to ping @tobybatch directly 😄

**Reply by derdeagle** [reply]
Thank you very much for your fast response. I'll do so!
